### PR TITLE
add a msg_box during the startup script to additional_apps

### PR DESCRIPTION
**Pull request by szaimen**
fix https://github.com/nextcloud/vm/issues/1565

### CODE REVIEW

File: menu/additional_apps.sh [hunk 1]
```diff
@@ -25,6 +25,13 @@ else
     STARTUP_SWITCH="OFF"
 fi
 
+# Show a msg_box during the startup script
+if [ -f "$SCRIPTS/nextcloud-startup-script.sh" ]
+then
+    msg_box "In the following menu will be additional apps offered to you that you can install.
+We have pre-selected apps that we recommend to any installation."
```
Comment (enoch85) [inline menu/additional_apps.sh:32]: Please have another go with the wording here.

Thanks!

File: menu/server_configuration.sh [hunk 2]
```diff
@@ -33,6 +33,13 @@ else
     STARTUP_SWITCH="OFF"
 fi
 
+# Show a msg_box during the startup script
+if [ -f "$SCRIPTS/nextcloud-startup-script.sh" ]
+then
+    msg_box "In the following menu will be server configuration offered to you that you can choose to configure.
+We have pre-selected configuration that we recommend to any installation."
```
Comment (enoch85) [inline menu/server_configuration.sh:40]: And here.

File: menu/additional_apps.sh (hunk 1 above)
Comment (enoch85) [reply menu/additional_apps.sh:32]: I can do it.

Comment (szaimen) [reply menu/additional_apps.sh:32]: > I can do it.

Would be great, thanks!

**Comment by enoch85** [comment]
Waiting for checks to pass, then I'll merge

**Pull request merged**